charts/eks-pod-identity-agent/values.yaml

```diff
@@ -37,6 +37,45 @@ daemonsets:
                   operator: In
                   values:
                     - hybrid
+  hybrid-bottlerocket:
+    create: false
+    nameSuffix: hybrid-bottlerocket
+    additionalArgs:
+      "--rotate-credentials": "true"
+    additionalSecurityContext:
+      seLinuxOptions:
+        type: spc_t
+    volumes:
+    - name: aws-credentials
+      hostPath:
+        path: /var/eks-hybrid/.aws
+        type: Directory
+    volumeMounts:
+    - mountPath: /root/.aws
+      name: aws-credentials
+      readOnly: true
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: "kubernetes.io/os"
+                  operator: In
+                  values:
+                    - linux
+                - key: "kubernetes.io/arch"
+                  operator: In
+                  values:
+                    - amd64
+                    - arm64
+                - key: "eks.amazonaws.com/compute-type"
+                  operator: In
+                  values:
+                    - hybrid
+                - key: os.bottlerocket.aws/variant
+                  operator: Exists
+                - key: os.bottlerocket.aws/version
+                  operator: Exists
```

Comment:
Shall we also update the existing helm chart for hybrid node to let it NOT run on bocketrocket OS https://github.com/aws/eks-pod-identity-agent/blob/main/charts/eks-pod-identity-agent/values.yaml#L36-L39

Reply:
Good call-out! Updated the PR, thanks